client/src/css/editor.css

```diff
@@ -0,0 +1,56 @@
+.orbeon__child-list {
+    margin: 1rem;
+}
+.orbeon__child-list .orbeon__child-list {
+    margin-right: -1rem;
+    padding-right: 0;
+}
+
+.orbeon__child-list-pages {
+    padding: 0.375rem 1rem;
+}
+
+.orbeon__child-list-item {
+    padding: 0.5rem 1rem;
+    border-bottom: 1px solid #aaa;
+    background-color: #eee;
+}
+.orbeon__child-list-item:last-child {
+    border-bottom: 0;
+}
+.orbeon__list-item-metadata {
+    display: flex;
+    align-items: center;
+}
+.orbean__child-toggle,
```

Comment:
I haven't tested running this yet, but it looks like a great start. However, the CSS class names definitely need tweaking. I think we should come up with something better than orbeon, and we should DEFINITELY not use inconsistent spellings of "orbeon" and "orbean" ;-) Can we just search-and-replace orbeon and orbean everywhere to objectEditor (or something like that)?

Reply:
We could go with just "editor"

Reply:
Works for me!